### PR TITLE
make emsdk installation instructions clear on which version need to be installed

### PR DESCRIPTION
**Pull request by pavelsavara**
As discussed here https://discord.com/channels/732297728826277939/732297825731215521/829090033856151665
Installing latest as per previous instructions could be incompatible with the current codebase.

### CODE REVIEW

File: src/mono/wasm/README.md [hunk 1]
```diff
@@ -4,16 +4,19 @@ This depends on `emsdk` to be installed.
...
 
 `EMSDK_PATH` is set to `$reporoot/src/mono/wasm/emsdk` by default, by the Makefile.
 
 Note: `EMSDK_PATH` is set by default in `src/mono/wasm/Makefile`, so building targets from that will have it set. But you might need to set it manually if
 you are directly using the `dotnet build`, or `build.sh`.
 
+* Alternatively you can install [expected version](./emscripten-version.txt) yourself from the [Emscripten SDK guide](https://emscripten.org/docs/getting_started/downloads.html).
+Do not install `latest` but rather specific version e.g. `./emsdk install 2.0.12`
+
+Make sure to set `EMSDK_PATH` variable, whenever building, or running tests for wasm.
```
Comment (am11) [inline src/mono/wasm/README.md:18]: +1, it used to have a default value of `$reporoot/src/mono/wasm/emsdk`, but not anymore.

Comment (pavelsavara) [reply src/mono/wasm/README.md:18]: I'm not sure what do you mean, could you please elaborate ?

Comment (am11) [reply src/mono/wasm/README.md:18]: Before `make -C src/mono/wasm provision-wasm; ./build.sh -os browser` used to work (without explicitly setting `EMSDK_PATH`) as the provision script installs it in a default (predictable) location. It doesn't work anymore; we have to set `EMSDK_PATH` in all cases.

File: docs/workflow/building/libraries/webassembly-instructions.md [hunk 2]
```diff
@@ -4,7 +4,10 @@
 
 If you haven't already done so, please read [this document](../../README.md#Build_Requirements) to understand the build requirements for your operating system.
 
-The Emscripten SDK (emsdk) needs to be installed.  Follow the installation guide [here](https://emscripten.org/docs/getting_started/downloads.html#sdk-download-and-install) or run `make -C src/mono/wasm provision-wasm` to install emsdk into `src/mono/wasm/emsdk`.
+The [expected version](..\..\..\..\src\mono\wasm\emscripten-version.txt) of Emscripten SDK (emsdk) needs to be installed.
+* Run `make -C src/mono/wasm provision-wasm` to install emsdk into `src/mono/wasm/emsdk`.
+* Alternatively follow the [installation guide](https://emscripten.org/docs/getting_started/downloads.html#sdk-download-and-install).
+Do not install `latest` but rather specific version e.g. `./emsdk install 2.0.12`
```
Comment (radekdoulik) [inline docs/workflow/building/libraries/webassembly-instructions.md:10]: You might mention https://github.com/dotnet/runtime/blob/main/src/mono/wasm/emscripten-version.txt here, it contains the current version used to build mono webassembly.

File: src/mono/wasm/README.md (hunk 1 above)
Comment (radekdoulik) [reply src/mono/wasm/README.md:18]: I think we are in the middle of transition to use nuget packages for emscripten and  I think this change is related to that. @radical knows the details.

File: docs/workflow/building/libraries/webassembly-instructions.md (hunk 2 above)
Comment (pavelsavara) [reply docs/workflow/building/libraries/webassembly-instructions.md]: improved

**Review by lambdageek — APPROVED** [review]

**Pull request merged**